client/extensions/woocommerce/app/settings/email/mailchimp/setup-steps/store-info.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormInputValidation from 'components/forms/form-input-validation';
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
+import LanguagePicker from 'components/language-picker';
+import SettingsPaymentsLocationCurrency from 'woocommerce/app/settings/payments/payments-location-currency.js';
+import TimeZone from 'components/timezone';
+import { translate } from 'i18n-calypso';
+
+const fields = [
+	{ name: 'store_name', label: translate( 'Store Name' ) },
+	{ name: 'store_phone', label: translate( 'Phone' ) },
+	{ name: 'admin_email', label: translate( 'Admin Email' ) },
+];
+
+const StoreInfo = ( { storeData, onChange, validateFields } ) => {
```

Comment:
Is there any chance `storeData` could be null, and calling `store_locale` and `store_timezone` could throw errors?

Reply:
Changing to isReuired and making sure in top level that we always have an object.

Reply:
Cool, maybe just to be extra safe `{ storeData = {}, onChange, validateFields }`